lib/Search/SearchQuery.php

```diff
@@ -95,7 +95,7 @@ public static function fromRequestArray(array $request)
     {
         $searchQuery = self::filterArray($request, 'search-query-string', '', FILTER_SANITIZE_STRING);
         $searchQueryAlt = self::filterArray($request, 'query_string', '', FILTER_SANITIZE_STRING);
-        $searchSize = self::filterArray($request, 'search-query-size', 10, FILTER_SANITIZE_NUMBER_INT);
+        $searchSize = self::filterArray($request, 'search-query-size', static::getDefaultSearchSize(), FILTER_SANITIZE_NUMBER_INT);
```

Comment:
Taking a quick look, would this not be better as a constant? that way we would avoid having a magic number while also allowing us to remove the: 
```php
public static function getDefaultSearchSize();
```
function in favour of using the class constructor since we would already have the ```private $size``` var and the getter function:
```php
    public function getSize()
    {
        if ((int)$this->size < 0) {
            $this->size = 1;
        }
        return (int)$this->size;
    }
```

Reply:
Yeah - a constant is more clear. 
I refactored to remove the static function. Initially, I hadn't realized that I could pass a null to the `filterArray` function. 

So I'm setting the default value in the constructor now. I still kept the code in it's own function in order to keep things short and specific.